assets/src/edit-story/elements/video/controls.js

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { useCallback, useEffect, useState } from 'react';
+import { useDebouncedCallback } from 'use-debounce';
+import { CSSTransition } from 'react-transition-group';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { ReactComponent as PlayIcon } from '../../icons/play.svg';
+import { ReactComponent as PauseIcon } from '../../icons/pause.svg';
+import StoryPropTypes from '../../types';
+
+const PLAY_BUTTON_SIZE = 48;
+const PLAY_ABOVE_BREAKPOINT_WIDTH = 108;
+const PLAY_ABOVE_BREAKPOINT_HEIGHT = 120;
+
+const Controls = styled.div`
+  position: absolute;
+  z-index: 2;
+  left: ${({ x }) => `${x}px`};
+  top: ${({ y }) => `${y}px`};
+  width: ${({ width }) => `${width}px`};
+  height: ${({ height }) => `${height}px`};
+`;
+
+const ButtonWrapper = styled.div.attrs({ role: 'button', tabIndex: -1 })`
```

Comment:
Is there a way to play/pause the video via keyboard as well?

Reply:
Not in [spec](https://docs.google.com/spreadsheets/d/17pZLd0faYZc9Xt7mG1KYtaZ0Krn5ppThIisz3TOsv8w/edit#gid=0) but probably it should after Beta? (space key to toggle the playback even if not hovered).

Reply:
Yep, post-beta should be OK, we can just add it to the post-beta todo list not to forget.